sqlx-cli/src/migrate.rs

```diff
@@ -30,7 +31,7 @@ fn create_file(
 
     let mut file = File::create(&path).context("Failed to create migration file")?;
 
-    file.write_all(migration_type.file_content().as_bytes())?;
+    std::io::Write::write_all(&mut file, migration_type.file_content().as_bytes())?;
```

Comment:
Had to fully qualify this due to the import of `std::fmt::Write` in order for the `write!` on string below to work